oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/hooks/useAxios.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+import { http } from "react-invenio-forms";
+
+// axios.defaults.baseURL = 'https://jsonplaceholder.typicode.com';
+
+export const useAxios = ({ url, method = "get", body = null }) => {
```

Comment:
Is it used anywhere?

Reply:
It is used in one place for the breadcrumbs component. As it will not work in general for us, I will leave it for now, until we resolve https://github.com/oarepo/oarepo-ui/issues/48. I will leave this issue open here.

Reply:
@Ducica that issue is now resolved in main

Reply:
@mirekys Ahoj Miro, we have a small issue. The languages are provided from BE in this form:

0
: 
{code: 'en', name: 'English'}
1
: 
{code: 'cs', name: 'čeština'}

But semantic ui drop down expects options to come in a bit different form and with a key property for mapping

 {
    key: 'Jenny Hess',
    text: 'Jenny Hess',
    value: 'Jenny Hess',
  },

Is this possible to change please or shall I manipulate it on the front end side?